test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1

```diff
@@ -339,4 +339,141 @@ Left Center Right
 			$output = $ps.Invoke()
 			$output.Replace("`n","").Replace("`r","") | Should -BeExactly $expected
 		}
+
+		It "Format-Table should correctly render headers that span multiple rows: <variation>" -TestCases @(
+			@{ view = "Default"; widths = 4,7,4; variation = "4 row, 1 row, 2 row"; expectedTable = @"
+
+Long Header2 Head
+Long         er3
+Head
+er
+---- ------- ----
+1    2       3
+
+
+
+"@ },
+			@{ view = "Default"; widths = 4,4,7; variation = "4 row, 2 row, 1 row"; expectedTable = @"
+
+Long Head Header3
+Long er2
+Head
+er
+---- ---- -------
+1    2    3
+
+
+
+"@ },
+			@{ view = "Default"; widths = 14,7,3; variation = "1 row, 1 row, 3 row"; expectedTable = @"
+
+LongLongHeader Header2 Hea
+                       der
+                       3
+-------------- ------- ---
+1              2       3
+
+
+
+"@ },
+			@{ view = "One"; widths = 4,1,1; variation = "1 column"; expectedTable = @"
+
+Long
+Long
+Head
+er
+----
+1
+
+
+
+"@ }
+		) {
+			param($view, $widths, $expectedTable)
+			$ps1xml = @"
```

Comment:
NIT: This is rather hard to read with large here strings in the middle of the code.

Reply:
In VSCode, the coloring makes it much easier to read.  Collapsing the XML will make it harder to read the XML.

Reply:
I agree too that it is more editors and formatting plugins issue and it strongly depends on personal preferences. We could be more happy with the support of YAML.